Formula/nqp.rb

```diff
@@ -14,7 +19,9 @@ class Nqp < Formula
     sha256 x86_64_linux:   "36ff4fcc84b071b261501dea0500acf9539cf631d4b6ffa2e5bb7dbdebf400e5"
   end
 
+  depends_on "libffi"
```

Comment:
Make this `uses_from_macos "libffi"` please. This will work as long as you don't need the absolute latest version of `libffi`. If you need 3.3+, you can add `, since: :catalina`.